lib/turn.rb

```diff
@@ -0,0 +1,24 @@
+#This is a file creating the turn class
+#ITERATION 1
+require './lib/card'
+
+class Turn
+    attr_reader :guess, :card
+
+    def initialize(guess, card)
+        @guess = guess
+        @card = card
+    end
+
+    def correct?
+        @guess == card.answer
+    end
+
+    def response
+        if correct? == true
```

Comment:
nice job using the method above so that you can keep your code DRY (Don't Repeat Yourself)